webpack/scenes/Subscriptions/UpstreamSubscriptions/UpstreamSubscriptionsPage.js

```diff
@@ -181,9 +181,9 @@ class UpstreamSubscriptionsPage extends Component {
     };
 
     const emptyStateData = () => ({
-      header: __('There are no Subscription Allocations to display'),
-      description: __('Subscription Allocations allow you to export subscriptions from the Red Hat Customer Portal to ' +
-        'an on-premise subscription management application such as Red Hat Satellite.'),
+      header: __('There are no Manifests to display'),
+      description: __('Manifests allow you to find, access, synchronize, and download content ' +
+        'from upstream Red Hat repositories for use in Satellite.'),
```

Comment:
Can you add Red Hat back in front of Satellite, according to the branding team when I was making kbase articles it needs to be referenced as `Red Hat Satellite`